src/checkers/inference/solver/util/PrintUtils.java

```diff
@@ -95,11 +94,11 @@ private static StringBuilder buildStatistic(Map<StatisticKey, Long> statistic,
             buildStatisticText(statistic, basicInfo, StatisticKey.GRAPH_SIZE);
         }
 
-        if (backEndType.getBackEndType().equals(BackEndTypeEnum.MaxSAT)
-                || backEndType.getBackEndType().equals(BackEndTypeEnum.Lingeling)) {
+        if (backEndType.fullyQualifiedName.equals(BackEndType.MAXSAT.fullyQualifiedName)
```

Comment:
String comparison is more expensive than enum comparison. Is there a special reason for using string comparison rather than directly compare the enum value?